rust/tests/data/golden/data-reader-array-primitives/_delta_log/00000000000000000000.json

```diff
@@ -0,0 +1,5 @@
+{"commitInfo":{"timestamp":1603724038064,"operation":"WRITE","operationParameters":{"mode":"ErrorIfExists","partitionBy":"[]"},"isBlindAppend":true}}
```

Comment:
I am thinking it's probably better to introduce these golden test dataset as a git submodule so we won't have to duplicate them into this repo. Eventually, we will be running our test suits against all the golden test tables all the time. But I am cool with us copying just this table to unblock the localstack fix for now and come back to change to submodule later.

Reply:
Agreed, I'm still trying to wear down @dennyglee to give me everything I want, including a separate golden data set only repo :)

Reply:
when that repo is ready, we also have couple new golden test data to contribute from @dispanser 's recent work.

Reply:
opened an issue https://github.com/delta-io/delta-rs/issues/179